paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/VerticalModeFormUIScreenshotTest.kt

```diff
@@ -47,6 +54,100 @@ internal class VerticalModeFormUIScreenshotTest {
         }
     }
 
+    @Test
+    fun fullCardForm() {
+        val metadata = PaymentMethodMetadataFactory.create()
+        val initialScreen = VerticalModeForm(
+            FakeVerticalModeFormInteractor.create(
+                paymentMethodCode = "card",
+                metadata = metadata,
+            )
+        )
+        val viewModel = FakeBaseSheetViewModel.create(metadata, initialScreen)
+
+        paparazziRule.snapshot {
+            ViewModelStoreOwnerContext {
+                PaymentSheetScreen(viewModel = viewModel, type = PaymentSheetFlowType.Complete)
+            }
+        }
+    }
+
+    @Test
+    fun fullCardFormWithLink() {
+        val metadata = PaymentMethodMetadataFactory.create()
+        val initialScreen = VerticalModeForm(
+            FakeVerticalModeFormInteractor.create(
+                paymentMethodCode = "card",
+                metadata = metadata,
+                canGoBack = false,
+            ),
+            showsWalletHeader = true,
+        )
+        val viewModel = FakeBaseSheetViewModel.create(metadata, initialScreen)
+        viewModel.walletsStateSource.value = WalletsState(
+            link = WalletsState.Link(
+                email = null,
+            ),
+            googlePay = null,
```

Comment:
Google pay was throwing errors. I could probably dig in more and get it to work, but it didn't seem high priority.

Reply:
Probably the button itself. There's probably network logic for figuring out whether to use the default button type or show your last 4 digits.

Reply:
The button is "Compose" but it really just wraps the original version that uses Android Views